lib/discordrb/data/member.rb

```diff
@@ -247,10 +247,10 @@ def server_unmute
     end
 
     # Bans this member from the server.
-    # @param message_days [Integer] How many days worth of messages sent by the member should be deleted.
+    # @param message_seconds [Integer] How many seconds worth of messages sent by the member should be deleted.
     # @param reason [String] The reason this member is being banned.
-    def ban(message_days = 0, reason: nil)
-      server.ban(@user, message_days, reason: reason)
+    def ban(message_seconds = 0, reason: nil)
```

Comment:
I would prefer if this breaking change happened in version 4.0, since it's such an easy thing to overlook if one doesn't expect a breaking change. For 3.x, I guess it would be better to support floating point numbers for `message_days`, multiply them with 86400 and turn that into an integer to pass on to the API.